src/services/userService.ts

```diff
@@ -16,35 +17,35 @@ class UserService {
     return user.save();
   }
 
-  public getUser(username: string) {
-    return User.findOne({ username }).exec();
+  public getUser(username: string, select = Select.default) {

```

Comment:
Ok, I see what you did here. In this case, the `Select` enum is better than const, as it doesn't allow to pass arbitrary string, but one of the "presets".